pkg/shortnames/shortnames.go

```diff
@@ -149,9 +157,9 @@ const (
 func (r *Resolved) Description() string {
 	switch r.rationale {
 	case rationaleAlias:
-		return fmt.Sprintf("Resolved short name %q to a recorded short-name alias (origin: %s)", r.userInput, r.originDescription)
+		return fmt.Sprintf("Resolve %q->%q (%s)", r.userInput, r.candidates(), r.originDescription)
```

Comment:
The origin is currently usually a path to a config file, which can contain both `unqualified-search-registries` and `aliases`, so this probably keep saying that it found an alias.